### PR TITLE
Mark CoreCLR packages stable

### PR DESCRIPTION
**Pull request by ericstj**
Getting this PR ready for when we want to push the button.

/cc @joshfree @weshaggard @schellap @gkhanna79

### CODE REVIEW

**Comment by schellap** [comment]
Targeting pack missing in stable versions.

**Comment by ericstj** [comment]
That's expected.  We don't want to publish that one to nuget.org since it's internal: right?

File: src/.nuget/Microsoft.NETCore.ILAsm/Microsoft.NETCore.ILAsm.pkgproj [hunk 1]
```diff
@@ -12,18 +12,13 @@
...
-      <Version>1.0.2$(VersionSuffix)</Version>
-    </Dependency>
+    <ProjectReference Include="..\Microsoft.NETCore.Runtime.CoreCLR\Microsoft.NETCore.Runtime.CoreCLR.pkgproj" />
     <ProjectReference Include="win\Microsoft.NETCore.ILAsm.pkgproj">
       <Platform>amd64</Platform>
     </ProjectReference>
     <ProjectReference Include="win\Microsoft.NETCore.ILAsm.pkgproj">
       <Platform>x86</Platform>
     </ProjectReference>
-    <ProjectReference Include="win\Microsoft.NETCore.ILAsm.pkgproj">
```
Comment (gkhanna79) [inline src/.nuget/Microsoft.NETCore.ILAsm/Microsoft.NETCore.ILAsm.pkgproj]: Please rebase as this change is now in release/1.0.0 from my PR.

**Comment by gkhanna79** [comment]
CC @pgavlin @russellhadley for JIT/ILAsm/ILDasm packages.

**Comment by pgavlin** [comment]
LGTM. Just so we know, what is the meaning of "stable" in this case?

File: src/.nuget/Microsoft.NETCore.ILDAsm/Microsoft.NETCore.ILDAsm.pkgproj [hunk 2]
```diff
@@ -12,18 +12,13 @@
...
-    </Dependency>
+    <ProjectReference Include="..\Microsoft.NETCore.Runtime.CoreCLR\Microsoft.NETCore.Runtime.CoreCLR.pkgproj" />
     <ProjectReference Include="win\Microsoft.NETCore.ILDAsm.pkgproj">
       <Platform>amd64</Platform>
     </ProjectReference>
     <ProjectReference Include="win\Microsoft.NETCore.ILDAsm.pkgproj">
       <Platform>x86</Platform>
     </ProjectReference>
-    <ProjectReference Include="win\Microsoft.NETCore.ILDAsm.pkgproj">
-      <Platform>arm</Platform>
```
Comment (gkhanna79) [inline src/.nuget/Microsoft.NETCore.ILDAsm/Microsoft.NETCore.ILDAsm.pkgproj]: As above.

File: src/.nuget/Microsoft.NETCore.Jit/Microsoft.NETCore.Jit.pkgproj [hunk 3]
```diff
@@ -18,9 +18,6 @@
     <ProjectReference Include="win\Microsoft.NETCore.Jit.pkgproj">
       <Platform>x86</Platform>
     </ProjectReference>
-    <ProjectReference Include="win\Microsoft.NETCore.Jit.pkgproj">
```
Comment (gkhanna79) [inline src/.nuget/Microsoft.NETCore.Jit/Microsoft.NETCore.Jit.pkgproj]: As above.

**Comment by ericstj** [comment]
Stable means the packages no-longer have a pre-release moniker.  This is what we will do right before we are ready to build for RTM.

**Comment by weshaggard** [comment]
:+1:

**Pull request merged**